Makefile

```diff
@@ -33,6 +33,7 @@ install:
 	/bin/cp -a static/* $(DESTDIR)
 	mkdir -p $(DESTDIR)/install-data
 	/bin/cp -r $(CRAFT_STAGE)/local-debs $(DESTDIR)/install-data/local-debs
+	$(CRAFT_PROJECT_DIR)/generate-connections.py $(CRAFT_PROJECT_DIR)/snap-connections.txt $(DESTDIR)/snap-connections.sh
```

Comment:
Could this not be created directly into /usr/libexec?

Reply:
Good point...

Reply:
I really left it there because it's just "for testing purposes", but you are right: it's better to keep it clean.

Reply:
Done.